packages/fluentui/accessibility/src/behaviors/Card/cardMultipleFocusableBehavior.ts

```diff
@@ -0,0 +1,40 @@
+import { Accessibility } from '../../types';
+import { CardBehaviorProps } from './cardBehavior';
+import { IS_FOCUSABLE_ATTRIBUTE, FocusZoneDirection } from '../..';
```

Comment:
Please avoid index imports

Reply:
It can be also a reason why this PR creates circular dependencies:

![image](https://user-images.githubusercontent.com/14183168/79442947-7bcec280-7fd9-11ea-86e1-7cc8ae3715e5.png)